pkg/compose/build_buildkit.go

```diff
@@ -76,3 +84,23 @@ func (s *composeService) doBuildBuildkit(ctx context.Context, opts map[string]bu
 
 	return imagesBuilt, err
 }
+
+func (s composeService) dryRunBuildResponse(ctx context.Context, options map[string]build.Options) (resp map[string]*client.SolveResponse, err error) {
+	buildResponse := map[string]*client.SolveResponse{}
+	for name, option := range options {
+		dryRunUUID := fmt.Sprintf("dryRun-%x", sha1.Sum([]byte(name)))
+		args := []string{"buildx", "build", "--print=outline", "--output=type=cacheonly", "-f", option.Inputs.DockerfilePath, option.Inputs.ContextPath}
+		buildCMD := exec.CommandContext(ctx, "docker", args...)
```

Comment:
we don't exec `docker buildx ...` for a standard build, but use the vendored buildx client, why not do the same here with `BuildOptions.PrintFunc`?

Reply:
As discuss internally, we'll wait for the buildx `--print` option to be marked as stable, this way we will only have to change the value in the `BuildOptions` and let buildx now when to build or not 👍 
I revert all the changes for a simplest solution for now